esmvaltool/diag_scripts/monitor/monitor_base.py

```diff
@@ -222,6 +222,7 @@ def record_plot_provenance(self, filename, var_info, plot_type, **kwargs):
             prov = self.get_provenance_record(
                 ancestor_files=[var_info['filename']],
                 plot_type=plot_type,
```

Comment:
After a quick `grep` it looks like about 50% of the diagnostics use `plot_type`, the other 50% `plot_types`. This is certainly not ideal, but probably not something we should enforce now. My recommendation would be to leave it like this and adapt your parser in https://github.com/ESMValGroup/ESMValCore/pull/2062 in such a ways that it is able to handle both cases.

Reply:
The documentation mentions only "plot_types", but I didn't expect it would be so common. My parser can already work with both, so I don't need it (hence why it's here as a suggestion); but it is still a good opportunity to update it.

Reply:
AFAIK, the monitoring functionalities are heavily used by BSC in production, so I really don't think it's a good idea to change that. This small change has the ability to break existing workflows.